docs/docs/07-cli.md

```diff
@@ -1814,9 +1814,9 @@ Interact with coordinator profiles
 
 **SEE ALSO**
 
-- [ignite chain](#ignite-chain) - Build, initialize and start a blockchain node or perform other actions on the
-  blockchain
-
+* [ignite network](#ignite-network)	 - Launch a blockchain in production
+* [ignite network coordinator set](#ignite-network-coordinator-set)	 - Set an information in a coordinator profile
+* [ignite network coordinator show](#ignite-network-coordinator-show)	 - Show a coordinator profile
```

Comment:
@aljo242 this file is generated by `scripts/gen-cli-docs` script, if you want to change its content you have to update the related code at `ignite/internal/tools/gen-cli-docs`.